pyaerocom/__init__.py

```diff
@@ -29,9 +29,7 @@
 try:
     # Opt in to new defaults introduced by xarray 2025.08.0
     # https://docs.xarray.dev/en/stable/whats-new.html#v2025-08-0-august-14-2025
-    xarray.set_options(
-        use_new_combine_kwarg_defaults=True, netcdf_engine_order=["netcdf4", "scipy", "h5netcdf"]
-    )
+    xarray.set_options(use_new_combine_kwarg_defaults=True)
```

Comment:
I think it is safer to keep the engine-order fixed. We have not excluded the v2025-8 release, and we haven't changed the comment.